components/BookCard/BookCard.js

```diff
@@ -57,12 +23,14 @@ export default function BookCard({
 
```

Comment:
we'll leave it as is for now because with destructuring we get a TypeError: Cannot destructure property 'isAlreadyRead' of 'booksInfo.find(...)' as it is undefined.